packages/playground/package.json

```diff
@@ -45,6 +45,12 @@
   },
   "homepage": "https://github.com/harness/canary",
   "dependencies": {
+    "@git-diff-view/react": "^0.0.16",
+    "reactivity-store": "^0.3.5",
+    "fast-diff": "^1.3.0",
+    "@git-diff-view/shiki": "^0.0.16",
+    "highlight.js": "^11.9.0",
+    "overlayscrollbars": "^2.10.0",
```

Comment:
Are all these libraries SecOps approved? We should also do performance analysis in case any of these libraries would end up bloating app bundle size.

Reply:
https://harness.atlassian.net/browse/SECOPS-14953

Reply:
https://harness.atlassian.net/browse/SECOPS-14955
https://harness.atlassian.net/browse/SECOPS-14954